ComputerInterface/BaseGameInterface.cs

```diff
@@ -87,6 +87,37 @@ public static ETurnMode GetTurnMode()
             return (ETurnMode) Enum.Parse(typeof(ETurnMode), turnMode);
         }
 
+        public static void SetInstrumentVolume(int value)
+        {
+            PlayerPrefs.SetFloat("instrumentVolume", (float)value / 50f);
+            PlayerPrefs.Save();
+        }
+
+        public static float GetInstrumentVolume()
+        {
+            float instVolume = PlayerPrefs.GetFloat("instrumentVolume", 0.1f);
+            return (instVolume);
+        }
+
+        public static void SetItemMode(EItemMode itemMode)
+        {
+            if (GorillaComputer.instance == null) return;
+
+            var itemModeString = itemMode.ToString();
+            PlayerPrefs.SetString("disableParticles", itemModeString);
+            PlayerPrefs.Save();
+            bool disableParticles;
+            if (itemModeString == "TRUE") {disableParticles = true;} else { disableParticles = false;}
+            GorillaTagger.Instance.ShowCosmeticParticles(!disableParticles);
+        }
+
+        public static EItemMode GetItemMode()
+        {
+            var itemMode = PlayerPrefs.GetString("disableParticles");
+            if (itemMode.IsNullOrWhiteSpace()) return EItemMode.TRUE;
```

Comment:
This is incorrect behavior, the default is false, see game code below
```cs
disableParticles = PlayerPrefs.GetString("disableParticles", "FALSE") == "TRUE";
```
```suggestion
            if (itemMode.IsNullOrWhiteSpace()) return EItemMode.FALSE;
```